app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1941,6 +1851,31 @@ class PatientRepositoryAndroidTest {
     assertThat(hasPatientChangedSince(patientUuid, oneSecondBeforePatientUpdated)).isFalse()
   }
 
+  @Test
+  fun patient_edits_should_not_affect_recent_patients() {
```

Comment:
Can you write similar test for Medical History as well?

Reply:
Sure. Added under `verify_recent_patients_are_retrieved_as_expected` test.